README.md

```diff
@@ -1,3 +1,5 @@
+# fedora Coreos Pipeline
```

Comment:
If you fix the capitalization (`Fedora CoreOS Pipeline`) I'll merge it.  :slightly_smiling_face:

Reply:
Done